validate_markdown_metadata.py

```diff
@@ -1,20 +1,100 @@
-from sys import argv
+import sys
+import yaml
+import re
 
-import frontmatter
+def validate_frontmatter(file_path):
+    try:
+        # Open the markdown file and extract content
+        with open(file_path, 'r') as f:
+            # Split the content by '---' which indicates YAML frontmatter in markdown
+            content = f.read().split('---')
```

Comment:
I completely agree with this. Just one little note:

I noticed that the `extract_yaml_and_body` from Portmap has a couple of things to store the **body** markdown. e.g. `body_content = []`

I'm not sure if I should keep it that way (I can always ignore it with something like this `frontmatter, _ = extract_yaml_and_body(content)` later on) since we need it to keep it consistent with Portmap or should I modify the method and delete those **body** functionalities?

Reply:
I think it makes sense to leave the method the same and just discard the body.  There's always a chance we'll want to add something to this validation code that does need the body.